ui/src/context/AnnotationStore.ts

```diff
@@ -97,6 +99,7 @@ export class PdfAnnotations {
     }
 
     withNewAnnotation(a: Annotation): PdfAnnotations {
+        console.log('Logging newly created annotation:', a);
```

Comment:
Do we always want to log this information? I find this very useful for debugging, but I'm not sure whether we want to commit this directly into the repository—perhaps we could add a line about this trick in the README?

Reply:
What do you think about adding a `deleteAnnotations` method to the class?

Reply:
Luckily, we already have a `delete annotation`. However, there are no class methods for performing bulk operations and we can add those. 

For example in this commit -
https://github.com/PaliwalSparsh/Form-A11y/pull/1/commits/16cdbf15937e0605928700daf80c954f190fb8f8

I had to write code for bulk deletion in the Selection component. Clearly, it could have been a part of the PDFAnnotations class.

Reply:
> Do we always want to log this information? I find this very useful for debugging, but I'm not sure whether we want to commit this directly into the repository—perhaps we could add a line about this trick in the README?

Oops! Something I did by mistake. No pre-commit hook to save me :P

I will remove this. However, this log is super useful for development.